### PR TITLE
fix: improve data sharing example

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

File: 03-terraform-outputs-across-stacks/stacks/vpc/ec2/main.tf [hunk 1]
```diff
@@ -16,10 +16,6 @@ data "aws_ami" "ubuntu" {
 
 resource "aws_instance" "web" {
   ami           = data.aws_ami.ubuntu.id
-  subnet_id     = data.terraform_remote_state.vpc.outputs.subnet_id
+  subnet_id     = data.terraform_remote_state.vpc.outputs.subnet.id
```
Comment (soerenmartius) [inline 03-terraform-outputs-across-stacks/stacks/vpc/ec2/main.tf:19]:
```suggestion
  subnet_id     = data.terraform_remote_state.vpc.outputs.subnet-main-a.id
```

**Review by soerenmartius — DISMISSED** [review]
tested the example, subnet output reference is wrong - other than that it works great :) cool approach

**Comment by mariux** [comment]
@soerenmartius can we continue here?

**Comment by soerenmartius** [comment]
> @soerenmartius can we continue here?

Yes, I have this on the agenda for this week still. Will get back

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**